### ISSUE TITLE
iu_xray

### ISSUE THREAD

**Issue by baoning1208** [Markin-Wang/XProNet#12]
Dear author, I apologize for disturbing you, but I sincerely believe that this is a great piece of work. However, I am not getting the desired results on the iu_xray dataset, and I feel that it might be due to issues with the hyperparameters. Could you please guide me on what the ideal hyperparameters should be? Thank you very much.

**Comment by Markin-Wang** [comment]
> Dear author, I apologize for disturbing you, but I sincerely believe that this is a great piece of work. However, I am not getting the desired results on the iu_xray dataset, and I feel that it might be due to issues with the hyperparameters. Could you please guide me on what the ideal hyperparameters should be? Thank you very much.

Hi, thanks for your interest, please refer to [issue 3](https://github.com/Markin-Wang/XProNet/issues/3) to see the details about this. Generally speaking, I would recommend you to tune the learning rate first. Note that IU-Xray is sensitive to the environment, hence the well-performed parameters may differ in different machines.
